core/platform/search/elastic_search_services.py

```diff
@@ -32,7 +32,7 @@
     cloud_id=feconf.ES_CLOUD_ID,
     http_auth=(
         (feconf.ES_USERNAME, feconf.ES_PASSWORD)
-        if feconf.ES_CLOUD_ID else None))
+        if feconf.ES_CLOUD_ID else None), timeout=60)
```

Comment:
The issue: https://github.com/oppia/oppia/issues/13199
exp_services.load_demo() takes more time than expected (since Release v3.0.8) increasing the timeout to unblock it

Reply:
Why is the timeout happening? We should document it here so that it's clear to others who come across this. Also, is this the desired behavior long-term? If not, we should have a TODO to revert it.

/cc @brianrodri

Reply:
@kevintab95 PTAL

Reply:
Found that this issue is appearing since the ElasticSearch was added for the dev server.

exp_services.load_demo() always needed more time to reload an exploration from local assets.
But when the ElasticSearch was added, it has set the default timeout of 10 seconds
which leads to the ReadTimeout Error I've mentioned.


Added a comment on why do we need this change
https://github.com/oppia/oppia/blob/89abd135bdcd0a93716de201223bf7b7cc9398d8/core/platform/search/elastic_search_services.py#L29-L31

/cc @seanlip

Reply:
That's a nice insight. Do we know what the underlying cause is for the ReadTimeoutError?

The timeout might be a reasonable adjustment given this, though it's not great to change a value like this for something that may only affect development or tests. That being said, I am concerned if ElasticSearch is introducing this much latency since that might indicate the possibility of performance regressions in production. @seanlip / @kevintab95 are either of you aware of this?

Reply:
Hm, I suspect this is just due to the handlers in question doing quite a lot of operations with e.g. saving many entities at once.

@kevintab95 have we seen any server errors related to this?

Reply:
Hm, I could not find any `ReadTimeoutError` errors.